t/17-basetest.t

```diff
@@ -319,10 +319,10 @@ subtest 'execute_time' => sub {
     my $mock_basetest  = Test::MockModule->new($basetest_class);
     my $test           = basetest->new('foo');
     is($test->{execution_time}, 0, 'the execution time is initiated correctly');
-    $mock_basetest->mock(run => sub { sleep 2; });
-    $mock_basetest->redefine(done => sub { });
+    $mock_basetest->mock(run => undef);
+    $mock_basetest->redefine(done => undef);
     $test->runtest;
-    is($test->{execution_time}, 2, 'the execution time is correct');
+    is($test->{execution_time}, 0, 'the execution time is correct');
```

Comment:
What I don't like about this change is, that we now never test if the `execution_time` field is touched after the initialization.
I would prefer to test for `>= 2`

Reply:
Sure, that would most likely work but not address that we waste two seconds just to test that two seconds have passed. Do you have another idea?

Reply:
ok, fixed with mocking the calculation of execution time. Please check again.

Reply:
Maybe we should log the time with milliseconds, at least if it is less than 1 second. Then we could `sleep 0.1`.

Reply:
Just looked into basetest.pm. We are using `Time::HiRes`, but don't import a function. So we could use its `time` function and use `sprintf "%.1f"` if it is smaler than 1 sec.

Reply:
Ok. Should we remove the `use Time::HiRes` also in this PR? It's not used.

Reply:
@perlpunk I was not sure about the Time::HiRes part so I merged after two other proposals to first fix the sporadic issue first. I hope that is ok for you. I will propose a PR for the remove of Time::HiRes after crosschecking other users or files.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/1407 for my followup